pkgs/applications/virtualization/qemu-test/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, runCommandCC, qemu, pkgconfig, ffmpeg_3, zlib }:
+
+(qemu.override {
+  hostCpuOnly = true;
+}).overrideAttrs (drv: {
```

Comment:
`overrideAttrs` considered harmful.

Reply:
Why? Can you elaborate on that?

Reply:
@edolstra: So would you instead recommend duplicating the Nix expression for QEMU? The attributes that are overridden here are only `patches` and `postPatch`, which should hardly break `qemu-test`. Apart from that even if it *does* break it, the fix should be easy enough. If we'd duplicate the expression an update on QEMU wouldn't break `qemu_test` but changes are high that `qemu_test` lags behind.